### DISCUSSION TITLE
标点符号问题

### DISCUSSION THREAD

**Discussion by Estheryan1** [sjtug/SJTUThesis#1046] [提问]
我用latex写的时候输入中文的？ ”“ 结果显示符号字体不是宋体
<img width="286" alt="截屏2024-06-30 13 48 26" src="https://github.com/sjtug/SJTUThesis/assets/148468519/cc88f11f-40c9-4db0-a8e5-ba07da8471d3">

**Comment by AlexaraWu** [comment]
华文宋体的问号就长这样，估计你是 macOS？可以装套 Windows 的中文字体，字体文件直接放到 `~/Library/Fonts` 里就行。然后设置选项 `cjk-font=windows`。

**Reply by Estheryan1** [reply]
cjk-font=windows 请问这个在哪里设置

**Reply by AlexaraWu** [reply]
在文档选项设置
```latex
\documentclass[cjk-font=windows]{sjtuthesis}
```
